src/SFML/Graphics/RenderTarget.cpp

```diff
@@ -53,6 +53,9 @@ namespace
             case sf::BlendMode::DstAlpha:         return GL_DST_ALPHA;
             case sf::BlendMode::OneMinusDstAlpha: return GL_ONE_MINUS_DST_ALPHA;
         }
+        assert(false);
+        sf::err() << "Invalid value for sf::BlendMode::Factor! Fallback to GL_ZERO." << std::endl;
```

Comment:
Position is alright. Optionally the error message could be copied to the assert, like:

`assert(false && "Blah blah")`

But I don't care as much, so :+1: :cat2: from my side.